week2/homework/src/index.js

```diff
@@ -1,3 +1,58 @@
-'use strict';
-
-// TODO: Write the homework code in this file
+'use strict';
+const program = require('commander');
+const handler = require('./handler.js');
+
+var todos = handler.read();
+
+program
+    .version('0.1.0');
+
+program
+    .command('help')
+    .description('usage of this cmd line program')
+    .action(function (param) {
+        program.help();
+    });
+
+program
+    .command('add [todo]')
+    .description('add to-do')
+    .action(function (param) {
+        handler.add(param);
+        console.log('\"' + param + '\" saved');
+    });
+
+program
+    .command('remove [index]')
+    .description('remove to-do by index')
+    .action(function (param) {
+        handler.remove(param);
+        console.log("removed by index: " + param);
+    });
+
+program
+    .command('list')
+    .description('list all to-dos')
+    .action(function () {
+        console.log(handler.read());

```

Comment:
It's hard to see what you're logging here, because you're logging a function call. I would do: 
```js 
const todos = handler.read();
console.log(todos);
```